src/tests/ftest/util/GeneralUtils.py

```diff
@@ -50,7 +51,46 @@ def get_file_path(bin_name, dir_path=""):
     else:
         return file_path
 
+def process_host_list(hoststr):
+    """
+    This utility function takes a slurm style host string and returns a list
+    of individual hosts.
+
+    e.g. boro-[26-27] becomes a list with entries boro-26, boro-27
+
+    This works for every thing that has come up so far but I don't know what
+    all slurmfinds acceptable so it might not parse everything possible.
```

Comment:
https://www.nsc.liu.se/~kent/python-hostlist/

Reply:
Now that it is done, just going to leave it.  Don't have to worry about license etc.

Reply:
Sure.  But if you find there are any defects in the implementation or it's not robust enough, it's probably worthwhile looking at the implementation above rather than continuing to expand on this one, IMHO.